include/overlay006/encounter_data.h

```diff
@@ -0,0 +1,33 @@
+#ifndef POKEPLATINUM_WILD_ENCOUNTER_DATA_H
+#define POKEPLATINUM_WILD_ENCOUNTER_DATA_H
+
+#include "overlay006/encounter_field_data.h"
+#include "overlay006/encounter_water_data.h"
+
+typedef struct WildEncounterData {
+    int encounterRateField;
+    FieldEncounterData fieldEncounters[12];
```

Comment:
Can you make some constants for these array lengths to make things easily configurable for a modder? e.g.,

```c
#define MAX_FIELD_ENCOUNTERS 12
#define MAX_SWARM_ENCOUNTERS 2
#define MAX_TIMED_ENCOUNTERS 2
#define MAX_RADAR_ENCOUNTERS 4
#define MAX_DUAL_SLOT_ENCOUNTERS 2
#define MAX_WATER_ENCOUNTERS 5
```